api/controllers/AccessTokenController.js

```diff
@@ -0,0 +1,22 @@
+const Promise = require('bluebird')
+
+module.exports = {
+  create: function (req, res) {
+    return LinkedAccount.tokenForUser(req.session.userId)
+    .then(account => account ? Promise.reject(new Error('User already has a token generated')) : Promise.resolve())
```

Comment:
small quibble -- you could do `account || Promise.reject(...)` instead of a conditional expression here